providerBuild.sh

```diff
@@ -253,9 +253,13 @@ if [ "$install_storage_support" = true ]; then
     helm repo update
     echo "Rook-Ceph repository added."
 
+    # Create the rook-ceph-operator.values.yaml file and append the kubelet directory location
+    touch /root/provider/rook-ceph-operator.values.yaml
+    echo "kubeletDirPath: /ephemeral/kubelet" >> /root/provider/rook-ceph-operator.values.yaml
```

Comment:
no need for `touch` here, since `>` (overwrite) or `>>` (append) will create the file anyway if it does not exist.

Also, if `providerBuild.sh` runs more than once - it will keep appending (`>>`) the `kubeletDirPath: /ephemeral/kubelet` line in `/root/provider/rook-ceph-operator.values.yaml` file.

Is providerBuild.sh supposed to be ran more than once? If so, then there should be a check if this line already present in the yaml and skip appending it; otherwise the yaml will have multiple of such lines.

Reply:
The providerBuild.sh script is supposed to be run once.